test/Microsoft.Health.Dicom.Web.Tests.E2E/Rest/UpdateInstanceTests.cs

```diff
@@ -101,10 +105,54 @@ public async Task GivenInstanceUpdated_WhenDeleting_ThenItShouldDeleteBothOrigin
         await VerifyDeleteStudyAsync(studyInstanceUid1, dicomFile1, requestOriginalVersion: true);
     }
 
-    private async Task UpdateStudyAsync(string studyInstanceUid, string expectedPatientName)
+    [Fact]
+    public async Task WhenUpdatingDicomMetadataWithExtendedQueryTagForASingleStudy_ThenItShouldUpdateCorrectly()
```

Comment:
This should be ok as these tests are in a different collection from the XQT ones. I wonder if perhaps XQT should have its own test collection.

Reply:
I felt this is just an extension of update. So kept it here, also I have other helper methods in the same so I dont want to duplicate.